controllers/default.py

```diff
@@ -71,9 +76,11 @@ def api():
     return Collection(db).process(request,response,rules)
 
 def register():
+    auth.settings.register_next = 'confirm.html'
     response.title = 'iDibo Registration'
```

Comment:
You could leave the title of the page hardcoded in the static html, since it's not changing (in views -> default -> register.html -> <title> tag)